ui/webui/src/components/installation/InstallationProgress.jsx

```diff
@@ -97,31 +121,45 @@ export class InstallationProgress extends React.Component {
         }
 
         return (
-            <Bullseye className={idPrefix + "-status-" + status}>
```

Comment:
This ID was used by the custom CSS to make icons having colors https://github.com/rhinstaller/anaconda/blob/master/ui/webui/src/components/installation/InstallationProgress.scss#L8

Please add a <div id={this id } wrapper for the styles to continue working.

Reply:
Interesting, should be fixed now. :)

Reply:
Or at least in theory, icon is still grey for some reason. :P